clients/githubrepo/graphql.go

```diff
@@ -34,6 +34,10 @@ const (
 	issueCommentsToAnalyze = 30
 	reviewsToAnalyze       = 30
 	labelsToAnalyze        = 30
+
```

Comment:
Using this comment to block merge in case it's reviewed today. Would rather merge this Monday so I can see what it does in the cron worker release test. If it got merged today, it would probably make its way to the production worker over the weekend